### ISSUE TITLE
The new mobile app synchronisation process fails (main issue: #7372;  test issue #7604)

### ISSUE THREAD

**Issue by BasilKaburi** [hzi-braunschweig/SORMAS-Project#8110]
The new mobile app synchronisation process that was introduced with 1.67  and continued on 1.68.0  was tested on the test performance server with a lot of data (#7266)
The expectation is that, synchronisation of entity data should occur in batches, however few.
Prerequisite: https://github.com/hzi-braunschweig/SORMAS-Project/issues/7303 and https://github.com/hzi-braunschweig/SORMAS-Project/issues/7521 have been merged into development and deployed to the test instance.

### Bug Description
No entity data synchronisation occurs following infrastructural synchronisation.
On user login, there is a flash of error message in red band which is too fast to allow reading and the case listing page opens with no cases.
### Steps to Reproduce
1. Create two surveillance officer user accounts A and B  for the district Voreingestellter Landkreis (it has 64,415 active cases and 115,487 contacts).
2.  Install SORMAS app on mobile device with sormas-1.69.0-SNAPSHOT-release.apk
3. Connect to the the performance server: https://test-performance.sormas.netzlink.com
4. Set up your testing device to use the full bandwidth available; log in with user A and verify that the initial synchronisation is completed successfully (i.e. there is no timeout, no error, and after logging in the data is shown in the lists). 
5. Document how long the sync took and how many cases and contacts where synced. Note: The initial sync also includes infrastructure. 
6. Log out from the app and set your testing device to limit bandwidth to Edge
7. Log in with user B, let the synchronisation run for 10 minutes and verify that the synchronisation is not cancelled by the app before (i.e. there is no timeout); cancel the synchronization after 10 minutes (because it would probably take a lot of time).
8. Document how many cases/contacts where synced. Note: When just switching the user, the infrastructure data will stay on the device.
9. Do another slow bandwidth sync for 5 minutes and document how many cases/contacts where synced

### Expected Behavior
At least one case should be synchronised at full bandwidth.

### Screenshots

![Screenshot_20220221-130122_FAST](https://user-images.githubusercontent.com/71414361/155337725-ddb3b034-0cd9-4640-9134-ec86e9a5b932.jpg)
![Screenshot_20220223-150900_Gmail](https://user-images.githubusercontent.com/71414361/155337838-f128b0ef-76d2-4304-93bc-2fc1323637cf.jpg)### System Details

*Device: Android
*SORMAS version: 1.68.0
*Android version/Browser: Version 11 (knox 3.7, API level 33)
*Server URL: https://test-performance.sormas.netzlink.com
*User Role: Surveillance officer & contact officer

**Comment by MateStrysewske** [comment]
@BasilKaburi @bernardsilenou Had another look at this issue. The behaviour you're experiencing is very likely correct. On a system with that many data, not even the person synchronization will be completed after 30 minutes or an hour, which means that case synchronization hasn't even started. 

I'm not sure whether some cases would appear when the sync process is aborted while cases are synchronized or whether there would be a database rollback, maybe @MartinWahnschaffe knows this.

If you really want to test the batch synchronization on a server with this amount of data, you will have to let it run until it completes (or fails) - aborting it manually will not produce reasonable results.

**Comment by BasilKaburi** [comment]
@MateStrysewske @MartinWahnschaffe 
Just to clarify, the initial synchronization aborts by itself anywhere between 15 to 35 minutes. This happened several times. The evidence I added to the issue is one such failed synchronization in 35th minute.
So your last recommendation: **"If you really want to test the batch synchronization on a server with this amount of data, you will have to let it run until it completes (or fails) - aborting it manually will not produce reasonable results"**
 has already been complied with.
Hence we did not create the issue based on manual abortion of the synchronization process.
It will be nice to know your findings on testing this issue.

**Comment by BasilKaburi** [comment]
@bernardsilenou @kwa20 FYI

**Comment by MateStrysewske** [comment]
@BasilKaburi Ah, okay, that wasn't clear to me based on the issue description. Can you please attach the logs that you sent to me to this issue as well so that the developer who has a look at it can use them?

**Comment by BasilKaburi** [comment]
@MateStrysewske Sorry my first description was not clear enough; you know I'm beginner on the job :)
Unfortunately, the size of the logs are unable to attach here so I will resend the email with the nubes link (and the other details of testing) to you to forward to developer, if you don't mind.
Given the wide variability of the time it takes for infrastructure data to sync, I strongly recommend that the developer performs several tests to observer the behaviour first hand. I hope this will give us some firm clues to nabbing the problem.
I'm also happy to have direct communication with the developer so we can exchange quick notes on the testing process.

**Comment by StefanKock** [comment]
Currently (with 1.74.x) there is no known sync issue anymore.
